src/pretix/plugins/stripe/static/pretixplugins/stripe/pretix-stripe.js

```diff
@@ -107,25 +112,65 @@ var pretixstripe = {
                             }
                         });
                         pretixstripe.card.mount("#stripe-card");
+                        pretixstripe.card.on('ready', function () {
+                            $('.stripe-container').closest("form").find(".checkout-button-row .btn-primary").prop("disabled", false);
+                        });
+                    }
+                    if ($("#stripe-sepa").length) {
+                        pretixstripe.sepa = pretixstripe.elements.create('iban', {
+                            'style': {
+                                'base': {
+                                    'fontFamily': '"Open Sans","OpenSans","Helvetica Neue",Helvetica,Arial,sans-serif',
+                                    'fontSize': '14px',
+                                    'color': '#555555',
+                                    'lineHeight': '1.42857',
+                                    'border': '1px solid #ccc',
+                                    '::placeholder': {
+                                        color: 'rgba(0,0,0,0.4)',
+                                    },
+                                },
+                                'invalid': {
+                                    'color': 'red',
+                                },
+                            },
+                            supportedCountries: ['SEPA'],
+                            classes: {
+                                focus: 'is-focused',
+                                invalid: 'has-error',
+                            }
+                        });
+                        pretixstripe.sepa.on('change', function(event) {
+                           if (['AD', 'PF', 'TF', 'GI', 'GB', 'GG', 'VA', 'IM', 'JE', 'MC', 'NC', 'BL', 'PM', 'SM', 'CH', 'WF'].includes(event.country)) {
```

Comment:
Can we add a comment where this list of countries comes from, how it is defined and under what conditions it might need to be updated?